spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-fabric8-client-loadbalancer/src/test/resources/wiremock-deployment.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     spec:
       containers:
         - name: servicea-wiremock
-          image: rodolpheche/wiremock:2.27.2
+          image: wiremock/wiremock:2.32.0
```

Comment:
the image that we use is deprecated, so just update to a newer, supported one (that has arm64 support also!)